### ISSUE TITLE
Add support for random modifiers on fields

### ISSUE THREAD

**Issue by rmurthy716** [open-traffic-generator/openapiart#307]
Can we support `choice: random` for all the different modifiers?

**Comment by ankur-sheth** [comment]
Right.  The other reason was that we needed all OTG implementations to generate the same stream of packets.  So 'random' for Ixia-c would have to generate the same 'random' for say TRex.  What are our use-cases for random?

**Comment by rmurthy716** [comment]
The use case is for load balancing hashing algorithms with random TCP ports. This is coming from a scan of a test plan shared with us.
